controllers/blogController.js

```diff
@@ -14,6 +14,21 @@ module.exports = {
       });
     }
   },
+  getOneBlog: async (req, res) => {
+    try {
+      const blogID = req.params.id;
+      const blog = await Blog.findOne({ _id: blogID });
```

Comment:
You could have used "findById" method instead of "findOne"

const blog = await Blog.findById(blogID);